lib/cli/options.js

```diff
@@ -54,16 +54,19 @@ const configuration = Object.assign({}, YARGS_PARSER_CONFIG, {
 
 /**
  * This is a really fancy way to:
- * - ensure unique values for `array`-type options
- * - use its array's last element for `boolean`/`number`/`string`- options given multiple times
+ * - `array`-type options: ensure unique values and evtl. split comma-delimited lists
+ * - `boolean`/`number`/`string`- options: use last element when given multiple times
  * This is passed as the `coerce` option to `yargs-parser`
  * @private
  * @ignore
  */
+const globOptions = ['spec', 'ignore'];
```

Comment:
Just checking, is `exclude` option alias'd into `ignore`?

Reply:
Yes, but vice versa. `ignore` is the canonical and `exclude` the alias option name. _Yargs-parser_ applies the coerce function automatically to both names.